tests/orm/test_querybuilder.py

```diff
@@ -753,6 +757,11 @@ def test_recreate_from_queryhelp(self):
         qb3 = copy.deepcopy(qb1)
         self.assertDictEqual(qb1.queryhelp, qb3.queryhelp)
 
+        qb1 = orm.QueryBuilder()
+        qb1.append(orm.Node)
+        qb2 = orm.QueryBuilder(**qb1.queryhelp)
+        self.assertDictEqual(qb1.queryhelp, qb2.queryhelp)
```

Comment:
Might as well just add an `append(Node)` statement to the very first querybuilder. Just add a comment that the `Node` one is important as it will indirectly test that `append` accepts `entity_type=''`

Reply:
Yeah, you are right. Since the original example was with the single append of `Node`, I was unsure that maybe when more appends where included then the internal logic may take a different path and not manifest this behavior, but I just checked this doesn't make a difference.